CHANGELOG.md

```diff
@@ -1,3 +1,83 @@
+# 2.0.0
+
```

Comment:
I am missing what is new in this version. The changelog should include that projects now need to filled in the admin site.

Reply:
I've now consolidated the data migration operations by type, going from safest to most risky in terms of potential data loss